app/javascript/components/workflows/professor/exams/new/editor/components/questions/Matching.tsx

```diff
@@ -58,254 +63,261 @@ const ChooseRightAnswer: React.FC<{
+      changeAllAnswers(index, index + 1);
+    }}
+    moveUp={(): void => {
+      fields.move(index, index - 1);
+      changeAllAnswers(index, index - 1);
+    }}
+    remove={(): void => {
+      fields.remove(index);
+      changeAllAnswers(index, -1, false);
+    }}
+  />
+);
+
+const OnePrompt: React.FC<{
+  memberName: string;
+  numChoices: number;
+  valueNum: number;
+  enableDown: boolean;
+  moveDown: () => void;
+  moveUp: () => void;
+  remove: () => void;
+}> = (props) => {
   const {
-    promptsLabel, prompts, valuesLabel, values,
-  } = info;
-  const [promptMoversVisible, rawSetPromptMoversVisible] = useState([]);
-  const setPromptMoversVisible = (index: number, visible: boolean): void => {
-    const newMovers = [...promptMoversVisible];
-    newMovers[index] = visible;
-    rawSetPromptMoversVisible(newMovers);
-  };
-  const [valueMoversVisible, rawSetValueMoversVisible] = useState([]);
-  const setValueMoversVisible = (index: number, visible: boolean): void => {
-    const newMovers = [...valueMoversVisible];
-    newMovers[index] = visible;
-    rawSetValueMoversVisible(newMovers);
-  };
-  const addPrompt = (): void => {
-    const newPrompts = [...prompts];
-    newPrompts.push({
-      type: 'HTML',
-      value: '',
-    });
-    onChange({ ...info, prompts: newPrompts }, value);
-  };
-  const deletePrompt = (index: number): UpdateBodyItemAction => {
-    const newPrompts = [...prompts];
-    newPrompts.splice(index, 1);
-    return makeChangeAction({ ...info, prompts: newPrompts }, value);
-  };
-  const movePrompt = (from: number, to: number): UpdateBodyItemAction => {
-    const newPrompts = [...prompts];
-    const fromOpt = newPrompts[from];
-    newPrompts.splice(from, 1);
-    newPrompts.splice(to, 0, fromOpt);
-    return makeChangeAction({ ...info, prompts: newPrompts }, value);
-  };
+    memberName,
+    numChoices,
+    valueNum,
+    enableDown,
+    moveDown,
+    moveUp,
+    remove,
+  } = props;
+  const [moversVisible, setMoversVisible] = useState(false);
+  return (
+    <FormSection name={memberName}>
+      <Row
+        className="p-2"
+        onMouseOver={(): void => setMoversVisible(true)}
+        onFocus={(): void => setMoversVisible(true)}
+        onBlur={(): void => setMoversVisible(false)}
+        onMouseOut={(): void => setMoversVisible(false)}
+      >
+        <Col className="flex-grow-01 pl-0">
+          <MoveItem
+            visible={moversVisible}
+            variant="dark"
+            enableUp={valueNum > 0}
+            enableDown={enableDown}
+            onUp={moveUp}
+            onDown={moveDown}
+            onDelete={remove}
+          />
+          {`${alphabetIdx(valueNum)}.`}
+        </Col>
+        <Col className="pr-0">
+          <Field
+            name="html"
+            component={EditHTMLField}
+            theme="bubble"
+            placeholder="Enter a new prompt"
+          />
+        </Col>
+        <Col sm={2}>
```

Comment:
This size isn't quite right:
![image](https://user-images.githubusercontent.com/918464/85142838-f05ef100-b216-11ea-8bf6-f92a4455af8f.png)

Reply:
I'm seeing this:
![image](https://user-images.githubusercontent.com/6677292/85147316-ae38ae00-b21c-11ea-848c-b32c0e44cb3a.png)